File: source/aoBlogs2/Models/MemberRuleModel.vb

Option Explicit On
Option Strict On

Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Contensive.BaseClasses

Namespace Models
    Public Class MemberRuleModel
        Inherits DbModel
        '
        '====================================================================================================
        '-- const
        Public Const contentName As String = "Member Rules"      '<------ set content name
        Public Const contentTableName As String = "ccMemberRules"   '<------ set to tablename for the primary content (used for cache names)
        Private Shadows Const contentDataSource As String = "default"             '<------ set to datasource if not default
        '
        '====================================================================================================
        ' -- instance properties

        Public Property DateExpires As Date
        Public Property GroupID As Integer
        Public Property MemberID As Integer
        '
    End Class
End Namespace

File: source/aoBlogs2/Models/Db/MemberRuleModel.cs

//using Contensive.Models.Db;
//using System;

//namespace Contensive.Addons.Blog.Models {
//    public class MemberRuleModel : Contensive.Models.Db.DbBaseModel {
//        // 
//        // ====================================================================================================
//        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("xxxxx", "xxxxx", "default", false);
//        // -- const
//        public const string contentName = "Member Rules";      // <------ set content name
//        public const string contentTableName = "ccMemberRules";   // <------ set to tablename for the primary content (used for cache names)
//        private  const string contentDataSource = "default";             // <------ set to datasource if not default
//        // 
//        // ====================================================================================================
//        // -- instance properties

//        public DateTime DateExpires { get; set; }
//        public int GroupID { get; set; }
//        public int MemberID { get; set; }
//        // 
//    }
//}